core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/scripts/model/RequisitionService.js

```diff
@@ -0,0 +1,69 @@
+/**
+* @author Alejandro Galue <agalue@opennms.org>
+* @copyright 2014 The OpenNMS Group, Inc.
+*/
+
+const _ = require('lodash');
+
+/**
+* @ngdoc object
+* @name RequisitionInterface
+* @module onms-requisitions
+* @param {Object} intf an OpenNMS interface JSON object
```

Comment:
Is it possible to use something more suitable in terms of naming?
Maybe `apiObject` or `opennmsApiObject`. 
`intf` sounds like a mix between integer and float and hurts in my head, while reading, as it just doesn't parse to something sane.

Reply:
I invite you to execute the following from the root of our repo:
```
git grep " intf"
```
That way you'll see the amount of places on which either an `IP Interface` or an `SNMP Interface` is referred using the name `intf` :)

Reply:
:joy: I would totally opt for using `intf`, too. But the funny thing is: nobody noticed, that the whole comment block is wrong and this is all about services, not interfaces. So, this would be `svc` now...

Reply:
That wasn't my point.
I interpreted the comment as `a generic api object from the OpenNMS Rest interface`, thus `apiObject` vs `interface`.